README.rst

```diff
@@ -1,4 +1,13 @@
-.. image:: https://codecov.io/gh/matrix-org/matrix-ios-sdk/branch/develop/graph/badge.svg?token=2c9mzJoVpu:target: https://codecov.io/gh/matrix-org/matrix-ios-sdk
+.. image:: https://img.shields.io/cocoapods/v/MatrixSDK?style=flat-square 
+   :target: https://github.com/matrix-org/matrix-ios-sdk/releases
+.. image:: https://img.shields.io/cocoapods/p/MatrixSDK?style=flat-square
+   :target: README.rst
```

Comment:
I needed to use a URL to get all the badges on the same line. This image with no URL is rendered within a `<p>` html block